flutter/lib/benchmark/benchmark.dart

```diff
@@ -531,27 +461,32 @@ enum DatasetMode { lite, full, test }
 
 class BenchmarkJob {
   final Benchmark benchmark;
-  final pb.DatasetConfig dataset;
+  late final pb.DatasetConfig dataset;
   final bool accuracy;
-  final bool fast;
-  final DatasetMode _datasetMode;
+  late final bool fast;
```

Comment:
`fast` can be private. 
Rename to `_fastMode` would be more meaningful. Please also rename `accuracy` to `accuracyMode`

Reply:
Done